docs/includes/apiargs-MongoDBCollection-method-mapReduce-option.yaml

```diff
@@ -13,6 +13,11 @@ name: finalize
 type: :php:`MongoDB\\BSON\\Javascript <class.mongodb-bson-javascript>`
 description: |
   Follows the reduce method and modifies the output.
+
+  .. note::
+
+     Passing a Javascript instance with a scope is deprecated. Put all scope
+     variables in the "scope" option of the MapReduce operation.
```

Comment:
I think our convention for referring to options uses monospace formatting instead of quotes, so we should write `scope` here. Feel free to double-check, though.

This made we wonder if the `scope` option for `mapReduce` was being preserved with the aggregation changes. According to the design document linked from [PM-766](https://jira.mongodb.org/browse/PM-766), it will remain available.

Reply:
Monospace formatting is the way to go in the rest of the docs, so I've adjusted it.

The deprecation message was suggested in the DRIVERS ticket, along with the "CodeWScope" wording which I decided to drop. This is why I assumed that the `scope` option would remain available. Thanks for confirming this.